js/twister_directmsg.js

```diff
@@ -5,216 +5,196 @@
 
 function requestDMsnippetList(dmThreadList) {
     var followList = [];
-    for( var i = 0; i < followingUsers.length; i++ ) {
+    for (var i = 0; i < followingUsers.length; i++)
         followList.push({username:followingUsers[i]});
-    }
-    for( var i = 0; i < groupChatAliases.length; i++ ) {
+    for (var i = 0; i < groupChatAliases.length; i++)
         followList.push({username:groupChatAliases[i]});
-    }
 
-    twisterRpc("getdirectmsgs", [defaultScreenName, 1, followList],
-           function(req, ret) {processDMsnippet(ret, dmThreadList);}, dmThreadList,
-           function(req, ret) {console.log("ajax error:" + ret);}, null);
+    twisterRpc('getdirectmsgs', [defaultScreenName, 1, followList],
+        function(req, ret) {processDMsnippet(ret, dmThreadList);}, dmThreadList,
+        function(req, ret) {console.log('ajax error:' + ret);}, null
+    );
 }
 
 function processDMsnippet(dmUsers, dmThreadList) {
     dmThreadList.empty();
 
-    for( var u in dmUsers ) {
-        if( dmUsers.hasOwnProperty(u) ) {
+    for (var u in dmUsers) {
+        if (dmUsers[u]) {
```

Comment:
are you sure it is safe to remove hasOwnProperty? i thought it was needed to avoid prototype's key-value pairs.

http://stackoverflow.com/questions/684672/loop-through-javascript-object